README.md

```diff
@@ -1,19 +1,36 @@
-<img src="banner.png" alt="Xamarin.iOS + Xamarin.Mac logo" height="145" >
-
-# Xamarin.iOS & Xamarin.Mac #
+# .NET for iOS, Mac Catalyst, macOS, tvOS #
 
 ## Welcome!
 
-This module is the main repository for both **Xamarin.iOS** and **Xamarin.Mac**.
+This module is the main repository for:
+- **.NET for iOS**
+- **.NET for Mac Catalyst**
+- **.NET for macOS**
+- **.NET for tvOS**
+
+These SDKs allow us to create native iOS, Mac Catalyst, macOS and tvOS applications using the same UI controls we would in Objective-C and Xcode, except with the flexibility and elegance of a modern language (C#), the power of the .NET Base Class Library (BCL), and two first-class IDEs&mdash;Visual Studio for Mac and Visual Studio&mdash;at our fingertips.
```

Comment:
Maybe mention VSCode instead of Visual Studio for Mac?

Reply:
done